.travis.yml

```diff
@@ -9,56 +9,48 @@ services:
 addons:
   postgresql: "9.5"
 elixir:
-  - 1.4.2
+  - 1.4.5
 otp_release:
   - 19.3
+notifications:
+  slack: 
+    rooms:
+      - secure: "bt5f6r5kl4XmmIeRN2JwB2N17QWz1Uj+KAOfW3UT7+r4+U7ggWFix+HuuZNJBjVzsvvHrPn/4L7uCLOSuSrBqt2mhPxJ3QGO/xVVCLI2luUawPFlm2AVRfRVvCUb9nx+YWpMSDAR1Qkzjl20mx3BxHeW54fnDw13lCV4EAzvsYVkA94HU6bDk9BWXolVrUggx26OdQ6TdzE3Y5ce841//q1VuzaFmlf9K9FWKFf7OlWPuToWNZj5PfiWNLiwMOYnTrt0shpNMT8HkdPKDHLFS1h8RHuimoOb3CLGyBWehgEK6tlAP7O9RZ5ECeezHbmsIAeZgZVQqhU/t2mSzGS9rRidl2hTeVcXW6EzLWZqi0VhUljjq2CFe3ZmWH3/Ql65hmvpJxPlsz3VhcELV6JGZGe9vpuqmK8KZWVqwj5H1MOwO6Kr29vnli6KwXGcDUGAvbMyWrjySWpmWs7AYrjCG7v5PVfy7vYPneLjuP/tgIivgsZFXRwANaozjVo03Dp17Yz/UpZ6DymwYtn7ZpjfHqLAFxw2ZN3azQXCD6N9jvehjIOInY+V+gYEqoI/M8H1qCu+Bd+nfSqR7doX7Pt+fQf/+gFy3dBWQTAOu4GwEkZEBlEhXAmLiWK5r2G5rxtm2XCgs3W/RnE4vYyCW8XHDPfKebtS92+pLrWahsBxhv4="
+    on_success: always
 env:
   global:
+    - Chart=mithril
     - MIX_ENV=test
-    - DOCKER_HUB_ACCOUNT=nebo15
+    - DOCKER_HUB_ACCOUNT=edenlabllc
```

Comment:
I guess you submitted a PR to a wrong repo 👯‍♂️

Reply:
sabotage